src/components/Restoration/RestorationTypeSummary.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <v-card flat id="restoration-type-summary" class="ma-4">
+
+      <div v-if="!isRestorationTypeValid" class="error-message pb-4">
+        <span>
+          <v-icon color="error">mdi-information-outline</v-icon>
+          <span class="error-text mx-1">This step is unfinished.</span>
+          <router-link
+            :to="{ path: `/${RouteNames.RESTORATION_BUSINESS_NAME}` }"
+          >Return to this step to finish it</router-link>
+        </span>
+      </div>
+
+      <v-row>
+        <v-col class="px-4">
```

Comment:
There's some unwanted leading whitespace here.

Compare this with https://github.com/bcgov/business-create-ui/blob/bf73ca09f6f81fe527689ac2b1c43ed3e1df8626/src/components/Incorporation/SummaryDefineCompany.vue#L21